content/en/blog/_posts/non-root-containers-and-devices.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: 'Non-root Containers And Devices'
+date: 2021-09-YY
+slug: non-root-containers-and-devices
+---
+
+**Author:** Mikko Ylinen (Intel)
+
+## Abstract
+One of the key security principles for running containers in Kubernetes is the
+principle of least privilege. The Pod/container `securityContext` specifies the config
+knobs to set, e.g., Linux* capabilities, MAC policies, and user/group ID values to achieve this.
+
+Furthermore, the cluster admins are supported with tools like [Pod Security Policy (deprecated)](docs/concepts/policy/pod-security-policy/) or
+[Pod Security Admission](docs/concepts/security/pod-security-admission/) to enforce the desired security settings for pods that are being deployed in
+the cluster. These settings could, for instance, require that containers must be `runAsNonRoot` or
+that they are forbidden from running with root's group ID in `runAsGroup` or `supplementalGroups`.
+
+However, the user/group ID related security settings trigger a problem when users want to
```

Comment:
Make this a part of the blog post purpose/goal at the beginning (see the below comment)